### PR TITLE
All services and Istio ingress configured in Kubernetes

### PR DESCRIPTION
**Pull request by gznznzjsn**

### CODE REVIEW

**Review by h1alexbel — CHANGES_REQUESTED** [review]
1st submission was misclicked

File: kubernetes/axon/axonserver-service.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+spec:
+  ports:
+    - name: gui
+      port: 8024
+      targetPort: 8024
+  selector:
+    app: axonserver
+  type: LoadBalancer
+  sessionAffinity: ClientIP
+---
```
[the remark this answers is not in the document]
Comment (gznznzjsn) [reply kubernetes/axon/axonserver-service.yaml:16]: My fault

File: kubernetes/run.sh [hunk 2]
```diff
@@ -26,3 +29,40 @@ kubectl apply -f task-service/task-service-configmap.yaml
...
+
+kubectl apply -f employee-service/employee-service-configmap.yaml
+kubectl apply -f employee-service/employee-service-secret.yaml
+kubectl apply -f employee-service/employee-service-deployment.yaml
+kubectl apply -f employee-service/employee-service-service.yaml
+
+# INGRESS
+kubectl apply -f istio-ingress.yaml
+
+minikube tunnel
```
[the remark this answers is not in the document]
Comment (gznznzjsn) [reply kubernetes/run.sh]: Done

File: kubernetes/employee-service/employee-service-secret.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: employee-service-secret
+type: Opaque
+data:
+  SPRING_DATASOURCE_USERNAME: cG9zdGdyZXM=
+  SPRING_DATASOURCE_PASSWORD: MXEydzNl
+  SPRING_DATA_MONGODB_USERNAME: dXNlcm5hbWU=
+  SPRING_DATA_MONGODB_PASSWORD: cGFzc3dvcmQ=
```
[the remark this answers is not in the document]
Comment (gznznzjsn) [reply kubernetes/employee-service/employee-service-secret.yaml]: Done

File: kubernetes/inventory-service/inventory-service-configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: inventory-service-configmap
+data:
+  SPRING_R2DBC_URL: 'r2dbc:postgresql://is-postgres-service:5432/postgres'
+  SPRING_JDBC_URL: 'jdbc:postgresql://is-postgres-service:5432/postgres'
+  SPRING_DATA_MONGODB_AUTHENTICATION-DATABASE: 'admin'
+  SPRING_DATA_MONGODB_PORT: '27017'
+  SPRING_DATA_MONGODB_HOST: 'is-mongo-service'
```
[the remark this answers is not in the document]
Comment (h1alexbel) [reply kubernetes/inventory-service/inventory-service-configmap.yaml:10]: fix the prefixes, SPRING -> APP, or something equal

**Review by h1alexbel — CHANGES_REQUESTED** [review]

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**